node-csv-parse.js

```diff
@@ -2,7 +2,7 @@
 // Created by Caleb Everett <everettcaleb95@gmail.com>
 // Licensed under GPL-2.0
 "use strict";
-!()=>{
+!(()=>{
```

Comment:
Might as well get rid of the exclamation point while you're at it :) Looks good, I'll accept the pull request and you can make that revision later if you feel the need to do so.